### ISSUE TITLE
MA0211 does not seem to apply to fields.

### ISSUE THREAD

**Issue by mjr4077au** [meziantou/Meziantou.Analyzer#1248]
### Version of the Meziantou.Analyzer NuGet package

3.0.134

### Rule Identifier

MA0211

### Target Framework

net472

### C# Language version

C# 14

### Description

From testing, only properties and enum values seem to pick up this rule. Fields of any kind, do not.

### Reproduction Steps

Minimal code:

```c#
/// <summary>Cloaks (set) the window so DWM composes it off-screen without presenting it.</summary>
public static int DWMWA_CLOAK { get; } = 13;  // Triggers MA0211

/// <summary>Cloaks (set) the window so DWM composes it off-screen without presenting it.</summary>
public static readonly int DWMWA_CLOAK = 13;  // Does not trigger MA0211

/// <summary>Cloaks (set) the window so DWM composes it off-screen without presenting it.</summary>
public const int DWMWA_CLOAK = 13;  // Does not trigger MA0211
```

**Comment by mjr4077au** [comment]
As a general thing, I'd expect this rule to apply on anything that can have a document comment. That'd include classes, structs, records, methods, etc also.
